### DISCUSSION TITLE
Can't find plugin to install

### DISCUSSION THREAD

**Discussion by claudiahuber2004** [intro-skipper/intro-skipper#420] [Q&A] [answered]
Hi all,

I added the repository as described here in the wiki:

[https://manifest.intro-skipper.org/manifest.json](https://manifest.intro-skipper.org/manifest.json)

Even after a restart of jellyfin there is no intro skipper plugin in the catalog.

Any idea why?

I use jellyfin in docker - the image is linuxserver/jellyfin 

Thx

**Comment by jumoog** [comment, marked as answer]
Do you have the latest version of jellyfin installed? (10.10
3) We don't support older versions

**Reply by claudiahuber2004** [reply]
Sorry for the late reply -
My version when I wrote this question was: 10.10.1

I just updated today and now I have version: 10.10.3 -> with this version the plugin shows up.

So it was indead a version issue.

Thank your for your feedback.

**Reply by bezdu** [reply]
> Do you have the latest version of jellyfin installed? (10.10 3) We don't support older versions

Sorry for reviving old thread.
I am having same issue with 10.11.7. 
I've added repository, but it won't show up in all plugins list. 

I have restarted the client via server and via docker - didn't help.

**Reply by AbandonedCart** [reply]
Update to 10.11.8

**Reply by bezdu** [reply]
I must be blind. I just checked and it was 10.11.7 on linuxserver... Sorry about that...

**Reply by iainb78** [reply]
Hi there! I'm new to JellyFin and having trouble getting the plug-in installed.

I'm on Jellyfin 10.11.9.

 I have installed the repository:

 and have rebooted Jellyfin multiple times but the intro-skipper app is just not showing up in the plugins library.

I'm probably missing something really dumb. Thanks!

(removed the screenshots to make the text more readable)

EDIT: I just updated to 10.11.10 and it's still not there.

**Reply by AbandonedCart** [reply]
Do you have a VPN installed? If so, does it work with it disabled?
Does the direct link to the manifest work? (https://raw.githubusercontent.com/intro-skipper/manifest/refs/heads/main/10.11/manifest.json)

**Reply by iainb78** [reply]
> Do you have a VPN installed? If so, does it work with it disabled? Does the direct link to the manifest work? (https://raw.githubusercontent.com/intro-skipper/manifest/refs/heads/main/10.11/manifest.json)

I do not have a VPN installed. I am trying to install on the computer that is running Jellyfin.

I made a new repository with the direct link and that is working.

**Reply by AbandonedCart** [reply]
Does it still work if you replace the url with `https://intro-skipper.org/manifest.json`
Make sure it is entered exactly as it appears.

If not, using the direct manifest is a viable workaround, but you will need to update the url for major version changes (10.11 -> 10.12, **not** 10.11.9 -> 10.11.10)

**Comment by iainb78** [comment]
I have neither repository installed and the plug in is stuck in the installed menu even through it's deleted...
<img width="1097" height="491" alt="image" src="https://github.com/user-attachments/assets/bf6b330c-2dc9-4652-ae79-263148b68cbc" />

EDIT: I re-added the repository and the plug-in shows as Deleted but it's also somehow enabled but the sidebar settings tab isn't there to adjust the plugin settings and the uninstall button doesn't work. 

<img width="820" height="346" alt="image" src="https://github.com/user-attachments/assets/482594c0-3352-45e9-85a2-fe625209c832" />

<img width="1217" height="619" alt="image" src="https://github.com/user-attachments/assets/2701a491-36f0-4f15-ac82-f62cd94efabd" />

**Reply by AbandonedCart** [reply]
You need to reboot the server. That's not a plugin issue. It's a server cache issue.

**Reply by iainb78** [reply]
I have restarted the server 5 times. Apparently it needed a full reboot.

It seems to be working now, with the original link that I had from the documentation:
https://intro-skipper.org/manifest.json

/shrug

Thanks for being so quick to help me!
